docs/getting-started/beginner-guide.md

```diff
@@ -1,85 +1,102 @@
 # <img class="dcr-icon" src="/img/dcr-icons/Flag.svg" /> Beginner's Guide
 
-Last updated for v1.0.1
+Last updated for Decrediton v1.2.1
 
 ---
 
 ## Welcome To Decred 
 
-Welcome to Decred! We are very glad to have you. As you may or may not already know, Decred is different. There's a lot of new things to learn to fully grasp how things work. This documentation will attempt to make things a bit easier to understand.
+This beginner's guide introduces Decred and offers signposts to some useful resources.
 
-This beginner's guide is a path to getting the Decred software up and running. You can choose from multiple clients to get a node on the network and Decred in your wallet. Specifically, you'll learn how to install, setup, and use the Decred applications; how to obtain DCR; how to vote via Proof-of-Stake; and how to use the block explorer.
+Decred is an autonomous digital currency. Stakeholders make and enforce the blockchain's consensus rules, set a course for future development, and decide how the project's treasury is used to fund it. Decred's blockchain is similar to Bitcoin's, but with major aspects of [governance](/governance/governance.md) baked into the protocol. 
 
-You'll need to follow an [Installation Guide](#installation-guides), and then follow the path for the application of your choice below. After your application is set-up, go through the [General Guides](#general-guides) at the bottom of this page. If you have any questions along the way, join us in our [Slack](/support-directory.md#join-us-on-slack).
+[Proof of Work](/mining/proof-of-work.md) miners play a similar role for Decred as they do for Bitcoin, but with Decred they only receive 60% of the block reward.
 
-**NOTE: Using Testnet is highly recommended for learning how to use the Decred software without the fear of making a mistake and losing real money. After following the guides to setup your choice of software, please visit the [Using Testnet guide](/getting-started/using-testnet.md) to learn how to launch your application onto Decred's testnet.**
+[Proof of Stake Ticket Voting](/mining/proof-of-stake.md) is central to Decred's governance. Decred holders can time-lock (or "stake") DCR to buy tickets. Tickets are randomly called to vote on-chain, this involves both approving the work of PoW Miners and voting Yes/No on any open [rule change proposals](/getting-started/user-guides/agenda-voting.md). 30% of the block reward goes to the holders of the tickets that voted in that block.
```

Comment:
To be honest, I think that the term "Proof of Stake Ticket Voting" might be too complex for beginners. Considering the sentence: 

> [Proof of Work](/mining/proof-of-work.md) miners play a similar role for Decred as they do for Bitcoin, but with Decred they only receive 60% of the block reward.

I suggest:

>[Proof of Stake](/mining/proof-of-stake.md) voting is central to Decred's governance. Decred holders can time-lock (or "stake") DCR to obtain voting tickets.

Note that I changed from "buy tickets" to "obtain voting tickets" because:
1. this immediately links "PoS voting" to the ticket concept
2. obtain is more neutral than buy (which implies spending)